kubeportal/api/views.py

```diff
@@ -80,7 +76,7 @@ def get_queryset(self):
             u = User.objects.get(pk=query_pk)
```

Comment:
Please wrap this in try/except and react on broken parameters with a proper HTTP response + log line.